.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
- [ ] Kindly use the `JavaScript + HTML + CSS linter` as per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=For%20this%20project%2C%20you%20should%20add%20the%20JavaScript%20%2B%20HTML%20%2B%20CSS%20linter.). Now the js linters is not running. You need to change `linters.yml` file with the new one in order to run.